DOCSP-16785 the the typo (#3297)
mongodb/docs-realm: temp/swift/users/custom-user-data.txt

Custom User Data - Swift SDK
============================

.. meta:: 

>   :description: Learn how to create custom user data using Atlas App Services in Swift.

Reviewer: in general we should always strive for consistency, e.g. the descriptions for similar pages across the SDKS (it's not critical for this PR since it will be resolved as part of the consolidation, but definitely something to keep in mind)
```suggestion
  :description: Learn how to work with custom user data using Atlas Device SDK for Swift.
```

>   :description: Learn how to work with custom user data using Atlas Device SDK for Swift.

Reviewer 2: ```suggestion
  :description: Learn how to work with custom user data using the Atlas Device SDK for Swift.
```
